### PR TITLE
Mobile-friendly homepage text

### PR DESCRIPTION
**Pull request by lancesnider**
I noticed a handful of issues with homepage text when viewing on mobile. I've annotated the changes in comments within the code so it's easier to see what changes go with what.

Here are the monster before and after screenshots: 
diff

Required Manual Testing
 
- [x] Does the non-logged in home page render correctly?
- [x]   Does the logged in home page render correctly?
- [x]   Does the projects page render correctly?
- [x]   Can you load project home pages?
- [x]   Can you load the classification page?
- [x]   Can you submit a classification?
- [x]   Does talk load correctly?
- [x]   Can you post a talk comment?

Review Checklist
  

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x]   Does it work on mobile?
- [x]   Can you rm -rf node_modules/ && npm install and app works as expected?
- [x]   If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x]   Are the tests passing locally and on Travis?

Optional

- [ ]   Have you replaced any ChangeListener or PromiseRenderer components with code that updates component state?
- [ ]   If changes are made to the classifier, does the dev classifier still work?
- [ ]   Have you resized and compressed any image you've added?
- [ ]   Have you added in flow type annotations?
- [ ]   Have you followed the Springer guidelines for commit messages?

### CODE REVIEW

File: css/home-page.styl [hunk 1]
```diff
@@ -437,7 +450,7 @@ $flex-section
     background-color: BACKGROUND_COLOR
     padding: 4em
 
-    @media (max-width: 700px)
+    @media (max-width: 768px)
```
Comment (lancesnider) [inline css/home-page.styl]: For consistency.

File: css/home-page.styl [hunk 2]
```diff
@@ -110,6 +112,10 @@ $flex-section
         padding: 0 10vw
         text-align: left
 
+      @media (max-width: 768px)
+        max-width: 100%
+        padding: 0 2em
+
```
Comment (lancesnider) [inline css/home-page.styl]: Text was too skinny: 

screen shot 2019-01-23 at 10 17 05 pm

The fix: 
screen shot 2019-01-23 at 10 18 21 pm

File: css/home-page.styl [hunk 3]
```diff
@@ -422,6 +432,9 @@ $flex-section
       margin: 2em 22vw
       align-items: stretch
 
+      @media (max-width: 768px)
+        margin: 2em
+
```
Comment (lancesnider) [inline css/home-page.styl]: The text was too skinny on mobile: 
screen shot 2019-01-23 at 10 11 18 pm

The fix: 
screen shot 2019-01-23 at 10 10 57 pm

File: css/home-page.styl [hunk 4]
```diff
@@ -567,6 +580,7 @@ $flex-section
       flex-direction: column
       margin-right: 3em
       max-width: 100%
+      padding-bottom: 5em
```
Comment (lancesnider) [inline css/home-page.styl]: The button was too close to the next block: 

screen shot 2019-01-23 at 10 29 56 pm

I set it to 5em to match the spacing of the spacing between other blocks on the page: 

screen shot 2019-01-23 at 11 05 46 pm

**Comment by lancesnider** [comment]
Looks like I'm duplicating some work that's already in progress (see https://github.com/zooniverse/Panoptes-Front-End/issues/5225#issuecomment-457354073). I'll close this and move onto something that's already been discussed in issues. :)

**Comment by srallen** [comment]
@lancesnider Apologies for not being clear in my comment! We're mainly focused on improvements to the classifier itself right now, with some other components being improved like the header and footer too. The updates here for the main homepage are still welcome and needed.

**Comment by lancesnider** [comment]
Okay, I've done all the required testing, though I haven't deployed a staging version. I'm assuming I need permissions for that.

**Comment by srallen** [comment]
@beckyrother are these style updates ok with you?

**Comment by beckyrother** [comment]
These look great! Thanks @lancesnider, this is really helpful.

**Pull request merged**